adal/src/main/java/com/microsoft/aad/adal/AcquireTokenSilentHandler.java

```diff
@@ -129,6 +130,24 @@ AuthenticationResult acquireTokenWithRefreshToken(final String refreshToken)
                 result.setRefreshToken(refreshToken);
             }
         } catch (final IOException | AuthenticationException exc) {
+            //if get 503,504,500 && outageMode is on, return the stale token
+            if (exc instanceof AuthenticationException
```

Comment:
Also why not catching the exceptions individually instead of catching them all together and then doing a instanceof?

---
In reply to: [70038804](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/676#discussion_r70038804) [](ancestors = 70038804)